src/os_view.eliom

```diff
@@ -68,6 +86,10 @@ let%shared connect_form ?a () =
         Form.string;
     ]) ()
 
+(** [disconnect_button ~a ()] creates a disconnect POST form with a button
+    without value, a signout icon and a text message "logout".
+
+    @param a modify the attributes of the form. *)
```

Comment:
Yes I agree. But I don't know what could be better. I though like `some additional attributes of the form`. @pwbs